### ISSUE TITLE
Duplicate entry

### ISSUE THREAD

**Issue by kwalcock** [WorldModelers/Ontologies#9]
It really seems like the entry
```
wm/concept/causal_factor/economic_and_commerce/economic activity/market/inflation
```
in wm_metadata.yml is duplicated.  In any case, it's tripping up the Eidos unit test.  Can someone verify and remove if that's the thing to do?

**Comment by kwalcock** [comment]
I can't assign anyone, so this is to bring it to @vanh17 's attention.  Thanks.  The change, if any, also needs to get back to Eidos.

**Comment by MihaiSurdeanu** [comment]
Yes, @vanh17, can you please fix this?

**Comment by vanh17** [comment]
Yes, I definitely can. I will check on this when I get off work today.

Best,
Henry

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by kwalcock** [comment]
I don't see that this has happened.  Are things OK, @vanh17?

**Comment by vanh17** [comment]
Hi Keith,

Sorry for the deplay. I have removed all the possible duplicates  Please let me know if the code is still broken.

Best,
Henry

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by kwalcock** [comment]
It seems to be better.  I was assuming that the change needed to be done to this WorldModelers/Ontologies repo, to which I have limited access, rather than directly to clulab/eidos.  That's why the call for help.  I guess that the change needs to be back ported now.  Eidos is happy now.  Thanks.

**Comment by BeckySharp** [comment]
in the future, we should make the changes here [WorldModelers/Ontologies]
first, as this is the "source"

**Comment by kwalcock** [comment]
Fixed with #10
